Stp.TestingApi/Controllers/TaskController.cs

```diff
@@ -308,7 +308,7 @@ public IActionResult UpdateTaskDescription(long taskId, [FromBody] string descri
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-        public ActionResult<List<SkillDto>> UpdateSkills(long taskId, [FromBody] List<SkillStateDto> skills)
+        public ActionResult<List<SkillDto>> UpdateSkills(long taskId, [FromBody] List<SkillStateDto?> skills)
```

Comment:
It is not correct to make SkillStateDto Nullable here. 
Do not change the type to Nullabe if there is no warning (like in this place).
Please review and fix other similar issues (if they exist).